managed/CounterStrikeSharp.API/Modules/Menu/CenterHtmlMenu.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System.Collections.Generic;
+using System.Text;
+
+namespace CounterStrikeSharp.API.Modules.Menu
+{
+    public class CenterHtmlMenuOption: IMenuOption
+    {
+        public Action<CCSPlayerController, IMenuOption> OnSelect { get; set; }
+        public string Text { get; set; }
+        public bool Disabled { get; set; }
+
+        public CenterHtmlMenuOption(string text, bool disabled, Action<CCSPlayerController, IMenuOption> onSelect)
+        {
+            Text = text;
+            Disabled = disabled;
+            OnSelect = onSelect;
+        }
+    }
+
+    public class CenterHtmlMenu: IMenu
+    {
+        public string Title { get; set; }
+        public List<IMenuOption> MenuOptions { get; } = new();
+
+        public CenterHtmlMenu(string title)
+        {
+            Title = title;
+        }
+
+        public IMenuOption AddMenuOption(string display, Action<CCSPlayerController, IMenuOption> onSelect, bool disabled = false)
+        {
+            var option = new CenterHtmlMenuOption(display, disabled, onSelect);
+            MenuOptions.Add(option);
+            return option;
+        }
+    }
+
+    public class CenterHtmlMenuInstance: BaseMenu
+    {
+        public CenterHtmlMenuInstance(BasePlugin plugin, CCSPlayerController player, IMenu menu) : base(player, menu)
+        {
+            plugin.RegisterListener<Core.Listeners.OnTick>(Display);
```

Comment:
I guess `OnTick` is necessary?

Reply:
It's how the previous guy implemented it CenterHtml Menus, this may also be required for `CenterMenu` too